src/WooCommerce.php

```diff
@@ -750,4 +769,58 @@ public static function getProductAttributes(\WC_Product $product) {
     return $data;
   }
 
+  /**
+   * Check if product gtin is found for any other product IDs.
+   *
+   * @param int $product_id Product ID.
+   * @param int $gtin GTIN.
+   * @return bool
+   */
+  public static function is_existing_gtin($product_id, $gtin) {
+    global $wpdb;
+
+    return $wpdb->get_var(
+      $wpdb->prepare(
+        "
+        SELECT pm.post_id
+        FROM {$wpdb->postmeta} pm
+        WHERE pm.meta_key = %s
+        AND pm.meta_value = %s
+        AND pm.post_id <> %d
+        LIMIT 1
+        ",
+        '_' . Plugin::PREFIX . '_gtin',
+        $gtin,
+        $product_id
+      )
+    );
+  }
+
+  /**
+   * Wrap is_existing_gtin for response to ajax requests.
+   *
+   * @param int $product_id Product ID.
+   * @param int $gtin GTIN.
+   * @return bool
+   */
+  public static function ajax_is_existing_gtin() {
+    $product_id = $_POST['product_id'];
+    $gtin = $_POST['gtin'];
+    $response = array(
+      'is_unique' => true,
+    );
```

Comment:
We are using [Drupal Code Standards to write arrays](https://www.drupal.org/docs/develop/standards/coding-standards#array).

So it should be:

```
$response = [
  'is_unique' => TRUE,
];
```